pkg/storegateway/storepb/rpc.proto

```diff
@@ -75,9 +75,10 @@ message SeriesRequest {
 
   // Thanos query_hints.
   reserved 12;
+}
 
-  // Thanos shard_info.
-  reserved 13;
+message Stats {
+  int64 index_bytes_fetched = 1;
```

Comment:
[nit] In `pkg/querier/stats/stats.proto` it's called `fetched_index_bytes`. I would keep the two consistent, so `fetched_index_bytes` (so that it's consistent with other stats too).